rfc/helm-import.md

```diff
@@ -0,0 +1,515 @@
+# Helm Import 
+
+* **Original Author(s):** @vinayak-kukreja
+* **Tracking Issue:** https://github.com/cdk8s-team/cdk8s/issues/1271
+* **API Bar Raiser:** @iliapolo
+
+Users are now able to import helm charts into their cdk8s app with `cdk8s import` command.
+
+## Working Backwards
+
+### README
+
+You can add a helm chart to your cdk8s application by running:
+
+```
+cdk8s import helm:https://airflow.apache.org/airflow@1.8.0
+```
+
+Here, `airflow` is the helm chart and `1.8.0` is the chart version.
+
+Then you can use this chart within your application:
+
+```typescript
+import { Construct } from 'constructs';
+import { App, Chart, ChartProps } from 'cdk8s';
+import { Airflow } from './imports/airflow';
+
+export class MyChart extends Chart {
+  constructor(scope: Construct, id: string, props: ChartProps = { }) {
+    super(scope, id, props);
+
+    new Airflow(this, 'airflow', {
+      values: {
+        fullnameOverride: 'Foo',
+      }
+    });
+  }
+}
+
+const app = new App();
+new MyChart(app, 'typescript-demo');
+app.synth();
+```
+
```

Comment:
Lets also add how the generated `airflow.ts` file will look like. This would make it clear that the values are typed.

Reply:
Makes sense, will add it.